### ISSUE TITLE
failures in tests with latest proj v4.9.1

### ISSUE THREAD

**Issue by jdkloe** [jswhit/pyproj#6]
Hi,
I just tried to build pyproj for Fedora Rawhide, using the brand new  proj-4.9.1 release and switched on the tests defined in pyproj/__init__.py.
I noticed that a fair number of them fail with this new proj version. Out of 73 tests the overall score was 54 passed and 19 failed (and it does not depend on the architecture used by the build.
Could you please take a look and let me know if this indicates real problems, or if this can be safely ignored for now?

The testresults are in these logfiles:
https://kojipkgs.fedoraproject.org//work/tasks/889/9210889/build.log
https://kojipkgs.fedoraproject.org//work/tasks/888/9210888/build.log
https://kojipkgs.fedoraproject.org//work/tasks/887/9210887/build.log

**Comment by jdkloe** [comment]
The log files have moved to a new location, you can see them now at:
http://koji.fedoraproject.org/koji/buildinfo?buildID=619820
https://kojipkgs.fedoraproject.org//packages/pyproj/1.9.4/1.fc23/data/logs/armv7hl/
https://kojipkgs.fedoraproject.org//packages/pyproj/1.9.4/1.fc23/data/logs/i686/
https://kojipkgs.fedoraproject.org//packages/pyproj/1.9.4/1.fc23/data/logs/x86_64/

**Comment by micahcochran** [comment]
The tests that are failing are the ones that require epsg, which are a commonly used as a shorthand for long projection string.  I would say that this is problematic.

**Comment by jdkloe** [comment]
thanks for your comment. Meanwhile I moved to release 2.0.0 for pyproj, and the test results for this one seem much better. All tests pass for python2 and only 5 out of 103 tests fail for python3. See: 
https://kojipkgs.fedoraproject.org//packages/pygrib/2.0.0/5.fc24/data/logs/x86_64/build.log
I'll try to find out what went wrong for python3 as soon as I find some time to work on it.

**Comment by jdkloe** [comment]
whoops, sorry, mixing two projects here. Version 2.0.0 refers to pygrib. Ofcourse pyproj is still at 1.9.4

**Comment by jdkloe** [comment]
pyproj still is at 54 passed and 19 failed, both for python2 and python3, see:
https://kojipkgs.fedoraproject.org//packages/pyproj/1.9.4/4.fc24/data/logs/x86_64/build.log
Next I'll check what happens with the latest version on github.

**Comment by jdkloe** [comment]
OK, latest github version looks good. All 73 tests pass both for python2 and python3. Closing this issues.

**Comment by jdkloe** [comment]
just for reference, the few failures reported by mistake above on pygrib v2.0.0, are also solved for the latest github version. Here both python2 and python3 versions give me 105 passed tests.

**Comment by micahcochran** [comment]
Hmm..  If the lastest github version passes all the test.  Perhaps there needs to be a new release.  Version 1.9.5?

**Comment by jdkloe** [comment]
yes, that would be nice. I noticed there are more reasons to go for a new release, see the discussion in issue #33 on Travis, so if that one is resolved it seems a good idea to me.

**Comment by jdkloe** [comment]
A final note on this issue. Afteral it seems the large number of fails was due to my own mistake in the building of the rpm package for pyproj.  In Fedora pyproj depends on proj but also on proj-nad and proj-epsg, and I had included these as requirements. However, they are also needed when running the unit tests during the rpm build process, and in this case they should be added as BuildRequires in the spec file as well. After adding this the majority of tests run fine. I'll mention the remaining failures (related to the hammer projection) in another issue.
